tests/src/Interop/Variant/VariantNative.cpp

```diff
@@ -0,0 +1,349 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include <xplatform.h>
+#include "platformdefines.h"
+
+#define LCID_ENGLISH MAKELCID(MAKELANGID(0x09, 0x01), SORT_DEFAULT)
+
+const BYTE NumericValue = 15;
```

Comment:
Bah. I really dislike this pattern of testing. It is how a lot of tests in the CLR codebase work, but it create a huge issue. The idea of putting the answer in two source locations is deeply problematic from a correctness perspective. Instead have the function take multiple variables for comparison or operation. For example look at the COM tests. Simple ones like add two numbers or reverse and return a string are perfect tests and require zero state coordination.

As a concrete example for the variant test. Pass in a value, verify its type as you are doing, double it and return. This will exercise a lot and make the native component very simple with 0 knowledge about what the right answer - which should _always_ be in the test itself not any assets.